lib/iex/endpoints/ref_data.rb

```diff
@@ -0,0 +1,19 @@
+module IEX
+  module Endpoints
+    module RefData
+      def isin_mapping(isins, options = {})
```

Comment:
The API is `asin`, why do we call it `asin_mapping`? Shouldn't it be `asin`?

Reply:
I called it as the documentation section was called (https://iexcloud.io/docs/api/#isin-mapping). As for me, it's a very controversial endpoint. It even doesn't return ISINs. It returns symbols that mutch to given ISINs.
Maybe it even would be better to call it `ref_data_isin`. What do you think?